src/v2/Components/AuctionResultPerformance.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import {
+  Text,
+  ThemeProviderV3,
+  IncreaseIcon,
+  Flex,
+  DecreaseIcon,
+  Spacer,
+} from "@artsy/palette"
+
+export const AuctionResultPerformance = ({
+  value,
+  align,
+}: {
+  value: string | null
+  align: "left" | "right"
+}) => {
+  if (value === null) {
+    return null
+  }
+
+  const sign = value[0] === "-" ? "down" : "up"
+  const color = sign === "up" ? "green100" : "red100"
+  const text = sign === "up" ? value : value.slice(1)
+  const Arrow = sign === "up" ? IncreaseIcon : DecreaseIcon
+
+  return (
+    <ThemeProviderV3>
+      <Flex
+        flexDirection="row"
+        justifyContent={align === "right" ? "flex-end" : undefined}
+        alignItems="baseline"
+      >
+        <Arrow fill={color} width={10} height={10} />
+        <Spacer mr="3px" />
```

Comment:
Stepping outside of the spacing scale should be avoided.

Reply:
Though I guess this is a lil weird cuz you're trying to have a natural space... 🤷  is fine

Reply:
Thats what i was thinking, since its kinda of like a text element

Reply:
I tried 0.3 and it didn't work 🤔🤔🤔. that's why I did 3px.

Reply:
intellisense... some day